### DISCUSSION TITLE
Daily 05/10

### DISCUSSION THREAD

**Discussion by marceloedelgado** [grupo-ispc-2023-2-0/workflix-app#171] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by BraianTroncoso** [comment]
Pude avanzar con el back, logrando la conexión a MySQL con springboot logrando tener comunicación correctamente con el servidor. Realicé con éxito el CRUD aplicando REST. Todavía no pushie nada porque tengo que hacer unas modificaciones asi queda generico.
Leyendo y viendo la clase de ayer que recomiendan SQLite para el uso de datos local, es posible utilizar ambas como se hace en un ámbito ya profesional, tener 2 bases de datos, si llegamos con el tiempo me gustaría que lo podramos incluir. 
Se complica un poco por la falta de tiempo libre y la alta carga horaria de muchas otras cuestiones externas, igualmente se avanza.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve revisando la plataforma, viendo videos de back y por la noche participé de la clase de inglés.
Hoy no tuve mucho tiempo. Estuve viendo como se hace la conexión de la base de datos y viendo que hay que implementar en nuestro proyecto. Voy a ver algunos videos más que pasaron por el grupo y a participar de la clase de hoy del instituto.

**Comment by Valentino399** [comment]
Ayer estuve practicando el lenguaje de java y hoy participe en la clase y hoy veré unos videos mas sobre java

**Comment by MarielaGomez** [comment]
Estuve repasando android con java con tutoriales y armando proyecto en paralelo, también tuve clases de inglés, estoy intentado comprender mejor la POO de Java, complica por el poco tiempo para lograr afianzar los conocimientos y también por carga de horario laboral externa. Ya pude ver la conexión con la base de datos. Seguiré practicando.
